README.md

```diff
@@ -65,13 +66,24 @@ Note that using `run.sh` sets these temporarily in the shell.
     
 ### Run OpenDroneMap
 
-First you need a set of images, taken from a drone or otherwise. Example data can be cloned from https://github.com/OpenDroneMap/odm_data
+First you need a set of images, taken from a drone or otherwise. Example data can be obtained from https://github.com/OpenDroneMap/odm_data
+
+Next, you need to copy over the settings file `default.settings.yaml` and edit it. The only setting you must edit is the `project-path` key. Set this to an empty directory within projects will be saved. There are many options for tuning your project. See the [wiki](https://github.com/OpenDroneMap/OpenDroneMap/wiki/Run-Time-Parameters) or run `python run.py -h`
```

Comment:
I understand this that way, you create settings.yaml in main directory of ODM, and sets `project-path` to `/directory_with_odm_projects/ `
Then starting ODM with `./run.sh odm_project_name `- this `--name` is optional here ;)

@pierotofy IMHO we need good documentation of OpenDroneMap anyway - in terms of API and using in production.